langchain4j-mcp/src/test/resources/logging_secured_mcp_server.java

```diff
@@ -0,0 +1,51 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//JAVA 21
```

Comment:
Need som guidance here to spin up secured map server

Reply:
You mean how to run this jbang script? If you have jbang installed, it should be just `jbang /path/to/script.java`

Reply:
Make secured quarkus mcp, with Bearer token123 as validation.